user-guide/modules/ROOT/pages/discussion-policy.adoc

```diff
@@ -0,0 +1,170 @@
+* Boost administration or policies
+* Compiler specific workarounds as applied to Boost libraries
+
+Other topics related to Boost development may be acceptable, at the discretion of moderators. If unsure, go ahead and post. The moderators will let you know.
+
+== Unacceptable Topics
+
+Posts on the following topics will probably not be accepted by the moderators:
+
+* Advertisements for commercial products
+* Requests for help getting non-Boost code to compile with your compiler (try https://stackoverflow.com/questions/tagged/c%2b%2b[Stack Overflow: C++])
+* Requests for help interpreting the pass:[C++] standard (try https://isocpp.org/faq[C++ FAQ])
+* Job offers
+* Requests for solutions to homework assignments
+* Any posting noted as <<Prohibited Behavior>>
+
+== Effective Posting
+
+Most Boost mailing lists host a great deal of traffic, so your post is usually competing for attention with many other communications. This section describes how to make sure it has the desired impact.
+
+=== Well-Crafted Posting is Worth the Effort
+
+Don't forget, you're a single writer but there are many readers, and you want them to stay interested in what you're saying. Saving your readers a little time and effort is usually worth the extra time you spend when writing a message. Also, Boost discussions are saved for posterity, as rationales and history of the work we do. A post's usefulness in the future is determined by its readability.
+
+=== Put the Library Name in the Subject Line
+
+When your post is related to a particular Boost library, it's helpful to put the library name in square brackets at the beginning of the subject line, e.g.
+
+```
+Subject: [Regex] Why doesn't this pattern match?
+```
+
+The https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers' mailing list] is a high-volume mailing list, and most maintainers don't have time to read every message. A tag on the subject line will help ensure the right people see your post.
+
+=== Don't Use Tabs
+
+If you use tabs to indent your source code, convert them to spaces before inserting the code in a posting. Something in the processing chain usually strips all the indentation and leaves a mess behind.
+
+=== Limit Line Length
+
+If you put source code in your postings and your mailer wraps long lines automatically, either keep the code narrow or insert the code as an (inline, if possible) attachment. That will help ensure others can read what you've posted.
+
+=== Don't Over-quote, Don't Top-Post, and Do Use Inline Replies for Readable Quotations
+
+Prune extraneous quoted text from replies so that only the relevant parts are included. It will save time and make your post more valuable when readers do not have to find out which exact part of a previous message you are responding to.
+
+Don't top-post (where the original message is included verbatim, with the reply above it); _inline_ replies are the appropriate posting style for Boost lists.
```

Comment:
It would be helpful to add a link to https://en.wikipedia.org/wiki/Posting_style?useskin=vector#Top-posting to describe what top-posting is.